src/package/packageConvert.ts

```diff
@@ -164,20 +162,21 @@ export async function createPackageVersionCreateRequest(
   const metadataZipFile = path.join(packageVersBlobDirectory, 'package.zip');
   const packageVersBlobZipFile = path.join(packageVersTmpRoot, 'package-version-info.zip');
 
-  const packageDescriptorJson = {
+  const packageDescriptorJson: PackageDescriptorJson & {
+    [prop: string]: unknown;
```

Comment:
is this really open ended?  Or is it only for the stuff in `copyDescriptorProperties` ?  In that case, we know what those types are.